src/puppet.js

```diff
@@ -653,6 +780,11 @@
     }
 
     this.queue.receive(this.obj, patches);
+    if(this.queue.pending && this.queue.pending.length && this.queue.pending.length > 1) {
```

Comment:
What about `3` ? ;)  at least we could refer in readme to complete explanation
https://youtu.be/xOrgLj9lOwk?t=1m29s